src/app/app.module.ts

```diff
@@ -15,9 +9,8 @@ import { ShowcaseComponent } from './showcase/showcase.component';
 import { TerraComponentsExamplesModule } from '../lib/terra-components-examples.module';
 import { RouterModule } from '@angular/router';
 
-export function initL10n(l10nLoader:L10nLoader):Function
-{
-    return ():Promise<void> => l10nLoader.load();
+export function initL10n(l10nLoader: L10nLoader): Function {
```

Comment:
ich habe mich sehr an die Klammern auf der nächsten Zeile gewöhnt aber sollte die Änderung nicht verhindern ;)

Reply:
Geht mir auch so, aber das ist (leider) auch nicht konfigurierbar. Andererseits ist die Klammerung von prettier auch irgendwie Standard in JS und TS